Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/1-index](https://hosted.weblate.org/projects/jamulus/1-index/).


It also includes following components:

* [Jamulus/Installation-for-Android](https://hosted.weblate.org/projects/jamulus/installation-for-android/)

* [Jamulus/Installation-for-iOS](https://hosted.weblate.org/projects/jamulus/installation-for-ios/)

* [Jamulus/Command-Line-Options](https://hosted.weblate.org/projects/jamulus/command-line-options/)

* [Jamulus/Installation-for-Linux](https://hosted.weblate.org/projects/jamulus/installation-for-linux/)

* [Jamulus/Contribution](https://hosted.weblate.org/projects/jamulus/contribution/)

* [Jamulus/Getting-Started](https://hosted.weblate.org/projects/jamulus/getting-started/)

* [Jamulus/Installation-for-Macintosh](https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/)

* [Jamulus/Installation-for-Windows](https://hosted.weblate.org/projects/jamulus/installation-for-windows/)

* [Jamulus/QOS-Windows](https://hosted.weblate.org/projects/jamulus/qos-windows/)

* [Jamulus/Running-a-Server](https://hosted.weblate.org/projects/jamulus/running-a-server/)

* [Jamulus/Client-Troubleshooting](https://hosted.weblate.org/projects/jamulus/client-troubleshooting/)

* [Jamulus/Custom-Directories](https://hosted.weblate.org/projects/jamulus/custom-directories/)

* [Jamulus/Include-Client-Commands](https://hosted.weblate.org/projects/jamulus/include-client-commands/)

* [Jamulus/Include-Server-Commands](https://hosted.weblate.org/projects/jamulus/include-server-commands/)

* [Jamulus/Include-Shared-Commands](https://hosted.weblate.org/projects/jamulus/include-shared-commands/)

* [Jamulus/Server-Bandwidth](https://hosted.weblate.org/projects/jamulus/server-bandwidth/)

* [Jamulus/footertext](https://hosted.weblate.org/projects/jamulus/footertext/)

* [Jamulus/Include-Backing-Up](https://hosted.weblate.org/projects/jamulus/include-backing-up/)

* [Jamulus/Privacy-Statement](https://hosted.weblate.org/projects/jamulus/privacy-statement/)

* [Jamulus/FAQ](https://hosted.weblate.org/projects/jamulus/faq/)

* [Jamulus/Server-Troubleshooting](https://hosted.weblate.org/projects/jamulus/server-troubleshooting/)

* [Jamulus/Software-Manual](https://hosted.weblate.org/projects/jamulus/software-manual/)

* [Jamulus/Tips-Tricks-More](https://hosted.weblate.org/projects/jamulus/tips-tricks-more/)

* [Jamulus/general](https://hosted.weblate.org/projects/jamulus/general/)

* [Jamulus/navigation](https://hosted.weblate.org/projects/jamulus/navigation/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/jamulus/-/1-index/horizontal-auto.svg)

melcon committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 7.4% (2 of 27 strings)

Translation: Jamulus/Installation-for-iOS
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-ios/pt_BR/

melcon committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 13.3% (2 of 15 strings)

Translation: Jamulus/Custom-Directories
Translate-URL: https://hosted.weblate.org/projects/jamulus/custom-directories/pt_BR/

melcon committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 3.9% (2 of 51 strings)

Translation: Jamulus/Client-Troubleshooting
Translate-URL: https://hosted.weblate.org/projects/jamulus/client-troubleshooting/pt_BR/

melcon committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 6.4% (2 of 31 strings)

Translation: Jamulus/Installation-for-Macintosh
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/pt_BR/

melcon committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 5.5% (2 of 36 strings)

Translation: Jamulus/FAQ
Translate-URL: https://hosted.weblate.org/projects/jamulus/faq/pt_BR/

melcon committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 4.0% (2 of 50 strings)

Translation: Jamulus/Installation-for-Windows
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-windows/pt_BR/

melcon committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 5.4% (2 of 37 strings)

Translation: Jamulus/Installation-for-Linux
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-linux/pt_BR/

melcon committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 1.4% (2 of 139 strings)

Translation: Jamulus/Running-a-Server
Translate-URL: https://hosted.weblate.org/projects/jamulus/running-a-server/pt_BR/

melcon committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 1.4% (2 of 135 strings)

Translation: Jamulus/Software-Manual
Translate-URL: https://hosted.weblate.org/projects/jamulus/software-manual/pt_BR/

melcon committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 4.2% (2 of 47 strings)

Translation: Jamulus/Tips-Tricks-More
Translate-URL: https://hosted.weblate.org/projects/jamulus/tips-tricks-more/pt_BR/

melcon committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 8.0% (2 of 25 strings)

Translation: Jamulus/Privacy-Statement
Translate-URL: https://hosted.weblate.org/projects/jamulus/privacy-statement/pt_BR/

melcon committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 10.0% (2 of 20 strings)

Translation: Jamulus/Installation-for-Android
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-android/pt_BR/

melcon committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 18.1% (2 of 11 strings)

Translation: Jamulus/QOS-Windows
Translate-URL: https://hosted.weblate.org/projects/jamulus/qos-windows/pt_BR/

melcon committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 28.9% (11 of 38 strings)

Translation: Jamulus/Getting-Started
Translate-URL: https://hosted.weblate.org/projects/jamulus/getting-started/pt_BR/

melcon committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 20.0% (2 of 10 strings)

Translation: Jamulus/Contribution
Translate-URL: https://hosted.weblate.org/projects/jamulus/contribution/pt_BR/

melcon committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 11.1% (2 of 18 strings)

Translation: Jamulus/Server-Bandwidth
Translate-URL: https://hosted.weblate.org/projects/jamulus/server-bandwidth/pt_BR/

melcon committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 18.1% (2 of 11 strings)

Translation: Jamulus/Command-Line-Options
Translate-URL: https://hosted.weblate.org/projects/jamulus/command-line-options/pt_BR/

melcon committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 7.1% (2 of 28 strings)

Translation: Jamulus/Server-Troubleshooting
Translate-URL: https://hosted.weblate.org/projects/jamulus/server-troubleshooting/pt_BR/

melcon committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 4.0% (1 of 25 strings)

Translation: Jamulus/navigation
Translate-URL: https://hosted.weblate.org/projects/jamulus/navigation/pt_BR/

trebmuh committed:
Translated using Weblate (French)

Currently translated at 100.0% (36 of 36 strings)

Translation: Jamulus/FAQ
Translate-URL: https://hosted.weblate.org/projects/jamulus/faq/fr/

jujudusud committed:
Translated using Weblate (French)

Currently translated at 100.0% (36 of 36 strings)

Translation: Jamulus/FAQ
Translate-URL: https://hosted.weblate.org/projects/jamulus/faq/fr/

jujudusud committed:
Translated using Weblate (French)

Currently translated at 96.2% (26 of 27 strings)

Translation: Jamulus/1-index
Translate-URL: https://hosted.weblate.org/projects/jamulus/1-index/fr/

trebmuh committed:
Translated using Weblate (French)

Currently translated at 95.6% (133 of 139 strings)

Translation: Jamulus/Running-a-Server
Translate-URL: https://hosted.weblate.org/projects/jamulus/running-a-server/fr/

trebmuh committed:
Translated using Weblate (French)

Currently translated at 91.4% (43 of 47 strings)

Translation: Jamulus/Tips-Tricks-More
Translate-URL: https://hosted.weblate.org/projects/jamulus/tips-tricks-more/fr/

trebmuh committed:
Translated using Weblate (French)

Currently translated at 100.0% (19 of 19 strings)

Translation: Jamulus/Include-Server-Commands
Translate-URL: https://hosted.weblate.org/projects/jamulus/include-server-commands/fr/

trebmuh committed:
Translated using Weblate (French)

Currently translated at 100.0% (135 of 135 strings)

Translation: Jamulus/Software-Manual
Translate-URL: https://hosted.weblate.org/projects/jamulus/software-manual/fr/

jujudusud committed:
Translated using Weblate (French)

Currently translated at 100.0% (37 of 37 strings)

Translation: Jamulus/Installation-for-Linux
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-linux/fr/

jujudusud committed:
Translated using Weblate (French)

Currently translated at 100.0% (50 of 50 strings)

Translation: Jamulus/Installation-for-Windows
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-windows/fr/

jujudusud committed:
Translated using Weblate (French)

Currently translated at 100.0% (20 of 20 strings)

Translation: Jamulus/Installation-for-Android
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-android/fr/

jujudusud committed:
Translated using Weblate (French)

Currently translated at 100.0% (27 of 27 strings)

Translation: Jamulus/Installation-for-iOS
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-ios/fr/

jujudusud committed:
Translated using Weblate (French)

Currently translated at 100.0% (31 of 31 strings)

Translation: Jamulus/Installation-for-Macintosh
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/fr/

jujudusud committed:
Translated using Weblate (French)

Currently translated at 100.0% (47 of 47 strings)

Translation: Jamulus/Tips-Tricks-More
Translate-URL: https://hosted.weblate.org/projects/jamulus/tips-tricks-more/fr/

trebmuh committed:
Translated using Weblate (French)

Currently translated at 100.0% (47 of 47 strings)

Translation: Jamulus/Tips-Tricks-More
Translate-URL: https://hosted.weblate.org/projects/jamulus/tips-tricks-more/fr/

jujudusud commented on _translator-files/po/fr/Tips-Tricks-More.po:
```diff
@@ -110,8 +110,8 @@ msgstr ""
-"**NOTE :** Cela ne les empêche pas de vous entendre, d'utiliser la fonction "
-"chat ou de voir les informations de votre profil.\n"
+"**NOTE :** cela ne les empêche pas de vous entendre, d'utiliser la fonction "
+"tchate ou de voir les informations de votre profil.\n"
```
Thanks, better like that!

jujudusud commented on _translator-files/po/fr/Tips-Tricks-More.po:
```diff
@@ -305,7 +305,7 @@ msgstr "## Pour les administrateurs de serveurs"
 msgid "### Converting a Registered Server to an Unregistered one on the fly"
 msgstr ""
-"### Convertir d'un serveur enregistré en serveur non enregistré à la volée"
+"### Convertir un serveur enregistré en serveur non enregistré à la volée"
```
Thanks. Nice like that.

ann0see commented on _translator-files/po/fr/1-index.po:
```diff
@@ -6,14 +6,16 @@
-"Last-Translator: \n"
-"Language-Team: \n"
+"PO-Revision-Date: 2022-08-15 15:17+0000\n"
+"Last-Translator: Julien Taverna <jujudusud@gmail.com>\n"
```
@jujudusud @ignotus666 Is it ok to have E-Mails here?
I'd rather not like to have mine in here

jujudusud replied:
I don't know if it is a good or a bad things.

ann0see replied:
I see it as a bit sub optimal (spam etc). My E-Mail can be found in my GH profile in encoded form, but we don't know if all contributors want their Mail being displayed here.

ann0see reviewed (commented):
Interesting.

ann0see committed:
Translated using Weblate (German)

Currently translated at 91.8% (34 of 37 strings)

Translation: Jamulus/Installation-for-Linux
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-linux/de/

ann0see committed:
Translated using Weblate (German)

Currently translated at 100.0% (31 of 31 strings)

Translation: Jamulus/Installation-for-Macintosh
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/de/

ann0see commented on _translator-files/po/de/Installation-for-Linux.po:
```diff
@@ -6,14 +6,16 @@
-"Last-Translator: \n"
-"Language-Team: \n"
+"PO-Revision-Date: 2022-08-15 20:17+0000\n"
+"Last-Translator: ann0see <ann0see@t-online.de>\n"
```
Yeah. As expected...

ann0see committed:
Translated using Weblate (German)

Currently translated at 94.9% (132 of 139 strings)

Translation: Jamulus/Running-a-Server
Translate-URL: https://hosted.weblate.org/projects/jamulus/running-a-server/de/

ignotus666 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (28 of 28 strings)

Translation: Jamulus/Server-Troubleshooting
Translate-URL: https://hosted.weblate.org/projects/jamulus/server-troubleshooting/es/

ignotus666 committed:
Translated using Weblate (Spanish)

Currently translated at 98.5% (133 of 135 strings)

Translation: Jamulus/Software-Manual
Translate-URL: https://hosted.weblate.org/projects/jamulus/software-manual/es/

ignotus666 committed:
Translated using Weblate (Spanish)

Currently translated at 97.8% (46 of 47 strings)

Translation: Jamulus/Tips-Tricks-More
Translate-URL: https://hosted.weblate.org/projects/jamulus/tips-tricks-more/es/

ignotus666 committed:
Translated using Weblate (Spanish)

Currently translated at 97.2% (35 of 36 strings)

Translation: Jamulus/FAQ
Translate-URL: https://hosted.weblate.org/projects/jamulus/faq/es/

ignotus666 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (51 of 51 strings)

Translation: Jamulus/Client-Troubleshooting
Translate-URL: https://hosted.weblate.org/projects/jamulus/client-troubleshooting/es/

ignotus666 committed:
Translated using Weblate (Spanish)

Currently translated at 94.2% (131 of 139 strings)

Translation: Jamulus/Running-a-Server
Translate-URL: https://hosted.weblate.org/projects/jamulus/running-a-server/es/

ignotus666 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (11 of 11 strings)

Translation: Jamulus/Command-Line-Options
Translate-URL: https://hosted.weblate.org/projects/jamulus/command-line-options/es/

melcon committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 63.1% (24 of 38 strings)

Translation: Jamulus/Getting-Started
Translate-URL: https://hosted.weblate.org/projects/jamulus/getting-started/pt_BR/

ann0see committed:
Translated using Weblate (German)

Currently translated at 100.0% (9 of 9 strings)

Translation: Jamulus/Include-Backing-Up
Translate-URL: https://hosted.weblate.org/projects/jamulus/include-backing-up/de/

ann0see committed:
Translated using Weblate (German)

Currently translated at 100.0% (20 of 20 strings)

Translation: Jamulus/Installation-for-Android
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-android/de/

ann0see committed:
Translated using Weblate (German)

Currently translated at 100.0% (27 of 27 strings)

Translation: Jamulus/Installation-for-iOS
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-ios/de/

ann0see committed:
Translated using Weblate (German)

Currently translated at 100.0% (31 of 31 strings)

Translation: Jamulus/Installation-for-Macintosh
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/de/

ann0see committed:
Translated using Weblate (German)

Currently translated at 91.8% (34 of 37 strings)

Translation: Jamulus/Installation-for-Linux
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-linux/de/

ann0see committed:
Translated using Weblate (German)

Currently translated at 98.0% (49 of 50 strings)

Translation: Jamulus/Installation-for-Windows
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-windows/de/

trebmuh committed:
Translated using Weblate (French)

Currently translated at 100.0% (27 of 27 strings)

Translation: Jamulus/1-index
Translate-URL: https://hosted.weblate.org/projects/jamulus/1-index/fr/

ann0see commented:
I'm not sure, but probably we could even merge this to release?

ignotus666 commented:
I think so. But maybe merge it into next-release first and then from there into release to avoid weird things happening with weblate's branch. It might not matter, but just in case.

ann0see commented:
No. The cleanest way is release->next release --> force update Weblate, I think.

ignotus666 commented:
Ok, that should work.

ann0see commented:
Hmm. No. There are some other unrelated changes on next-release. So I'll merge it to next-release.

ann0see commented:
I'll take care of the E-Mail tomorrow manually.

ignotus666 commented:
I wonder why it still says there are 52 outgoing commits on weblate - it should have been reset to zero, shouldn't it?

ann0see commented:
It needs to be force reset.

ann0see commented:
Done

ignotus666 commented:
Will it always have to be forced? I didn't have to do that in my tests.

ann0see commented:
Only if we squash the commits. It was certainly needed here, since there were way too many small commits.

ann0see commented:
I think fixing conflicts on the weblate repo is more difficult. I now did it on GitHub the standard way.
